uftrace.c

```diff
@@ -1072,6 +1092,9 @@ int main(int argc, char *argv[])
 	if (opts.use_pager && opts.mode != UFTRACE_MODE_LIVE)
 		start_pager(pager);
 
+	/* apply 'uftrace.opts' options for analysis commands */
+	apply_uftrace_opts(&argc, &argv, &opts);
```

Comment:
Are you sure about the priority of option?  I mean command line option should override one in the `uftrace.opts`.  It'd be nice to have a test case to verify it works correctly.

Reply:
You're right. I have to move this in a different place.

Reply:
I added a test if it doesn't override user setting time-filter.